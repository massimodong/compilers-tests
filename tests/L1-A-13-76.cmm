struct F_6OsFqbmYxrei2I57 {
}* myetWOEZfCssBHX ( ) {
  float fbrHepCIiwEmtFLC <= ;
}
int ;
struct XqPjiyEA5 ;/ 
int qM5fmwR9s6 ~( int A6pBHolNFO4bi0ifzZ ) {
  if ( dgGnwR6ns = cgGzL3 || tYkySCP2nUZ1BjaQ6P != 
    W - ymaeCQW_dMk0l / ( H6z5qO && cSsBrmEq > zNBNqebaNtaH9Quy 
    + n31dGGu0n7UiusWJ_mWw * - ! xFd [ Qg1D8Z_Y85iTHPKO3 
    == ! CXmFd ( zN85LwLuueEN1MJnKVw && qP3HbBnmnIcS5 
    * - [ r && Zq7yVpm53D8 >= ! - cHXyUA [ oJTKF <= ! 
    P9 [ aT != koDQxxrQY + Wz2U_6ZC7XkHiVL [ V = JVdjp_fn2Aef 
    && _1_Pzn0jjju_ - huHUKC0Ytli_Haalz3 / - - - PEzKzD1TmbYpaiRU3 
    ( ZaxCWV61vRHibhmyzi8 , v8OOYT6kUuNWWXFRmNk ) . Z3mtWL83PoSPnxGQ 
    < GyCusO ] - - ( l != n0Q <= R9R ) * 1531754999 ] 
    ] + 1983414071.1463453357 - B7i6EL ] . eToev6E3tDE8MLcRw4p 
    . KVFtoHUok7Bd1sfR < AEtCpHyMbuvKTf2wQs6f ( ) ) ] 
    ) ) {
  }
}
struct K8T8skjOtOE Ji_cQRij5pd_XzlPOEN6 [ :1398204582 
  ] 
;
int DB5U9 ( int lEd2_0JRHgd , int bj ) {
  if eNDWSW_k2cTlXr )   C7sh ;
else   if ( &rTdNmKSeLc )   qXGjW9HDGs2H ;


  HWlY ;
  O_3j_smxjZQj ;
  if ( jDODzPHtInRE3iRyJO_ )   while ( WUR3pX8u1M72bdJK8Vk 
    )   return lzj_nsNJtHAow ;

else   mTz2cHbpf8JbK 

}
int ;
int $zgkcPeZ4xCarkEvNa ( ) {
}
struct {
  int GfzdQY_KMSeaC5 , FXMIdzkPtckx , fY8JBNJQ86J 
    = w5dTSLVNsJ9nU ;
}IieQ3XrAR0dI , hQVc 
`
;
int ;
struct ;
