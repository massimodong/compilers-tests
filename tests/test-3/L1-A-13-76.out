Error Type B at Line 2
Error Type B at Line 3
Error Type B at Line 6
Error Type A at Line 7
Error Type B at Line 12
Error Type A at Line 22
Error Type B at Line 26
Error Type A at Line 27
Error Type B at Line 37
Error Type A at Line 39
Error Type A at Line 45
Error Type B at Line 48
